app/models/setting/auth.rb

```diff
@@ -21,8 +21,8 @@ def self.default_settings
       set('websockets_encrypt', N_("VNC/SPICE websocket proxy console access encryption (websockets_ssl_key/cert setting required)"), !!SETTINGS[:require_ssl], N_('Websockets encryption')),
       set('login_delegation_logout_url', N_('Redirect your users to this url on logout (authorize_login_delegation should also be enabled)'), nil, N_('Login delegation logout URL')),
       set('authorize_login_delegation_auth_source_user_autocreate', N_('Name of the external auth source where unknown externally authentication users (see authorize_login_delegation) should be created (If you want to prevent the autocreation, keep unset)'), 'External', N_('Authorize login delegation auth source user autocreate')),
-      set('authorize_login_delegation', N_("Authorize login delegation with REMOTE_USER environment variable"), false, N_('Authorize login delegation')),
-      set('authorize_login_delegation_api', N_("Authorize login delegation with REMOTE_USER environment variable for API calls too"), false, N_('Authorize login delegation API')),
+      set('authorize_login_delegation', N_("Authorize login delegation with REMOTE_USER HTTP header"), false, N_('Authorize login delegation')),
+      set('authorize_login_delegation_api', N_("Authorize login delegation with REMOTE_USER HTTP header for API calls too"), false, N_('Authorize login delegation API')),
```

Comment:
@ekohl are you ok with this change in the description?
@hsahmed you'll need to create a migration to update the description for existing settings, should be something along the lines of:

```ruby
Setting.where(name: 'authorize_login_delegation').update_all(description: "Authorize login delegation with REMOTE_USER HTTP header")
Setting.where(name: 'authorize_login_delegation_api').update_all(description: "Authorize login delegation with REMOTE_USER HTTP header for API calls too")

Reply:
Creating a file in db/migrate with a timestamp prefix using format +%Y%m%d%H%M%S with the following content would suffice?
```
class UpdateLoginDelegationDescription < ActiveRecord::Migration[6.0]
  def up
    Setting.where(name: 'authorize_login_delegation').update_all(description: "Authorize login delegation with REMOTE_USER HTTP header")
    Setting.where(name: 'authorize_login_delegation_api').update_all(description: "Authorize login delegation with REMOTE_USER HTTP header for API calls too")
  end
end
```

Reply:
you can use `rails generate migration UpdateLoginDelegationDescription` to automatically create an empty file for you. you can also add a `down` migration to change the description back to the previous version just for completeness sake.